### PR TITLE
Update content_how_to_get_compensated_for_work.md

### PR DESCRIPTION
**Pull request by vsterling**
Requesting updates to the contents of How to Get Compensated with relevant information.

### CODE REVIEW

File: en/content_how_to_get_compensated_for_work.md [hunk 1]
```diff
@@ -1,47 +1,71 @@
...
-SCRF pays contributors by the hour. Hours are tracked in [a spreadsheet](https://docs.google.com/spreadsheets/d/19CmaqELKS_S2gWQIs2kddA91rv5zca-51mk_sOJ0gYs/edit?usp=sharing) that is submitted once a month with your invoice (see below). Please share a copy of this sheet with your team lead.
+Anyone submitting bounties or non-hours-based grants can ignore this step. Anyone else working for SCRF is expected to track their hours using the template sheet and process outlined below:
 
-## Invoicing and Payment System
+* Make a copy of the [Hours Template Sheet](https://docs.google.com/spreadsheets/d/1pXZlO_mxAbKQ-0gifylb2chdUuU3OniBWllryJ48RAM/edit?usp=sharing) and title it '[Your Name] Hours'
+* Track time worked in 30-minute increments on the spreadsheet copy
+* Provide a detailed description of work completed for each line item
+* Submit the template at the end of the month with an invoice
+
+## Compensation & Logistics
```
Comment (ghettodev) [inline en/content_how_to_get_compensated_for_work.md:44]: We should pick ampersands or 'and' in the headers and not alternate.

File: en/content_how_to_get_compensated_for_work.md [hunk 2]
```diff
@@ -1,47 +1,71 @@
...
+1. Choose DAI for invoice currency.
+2. Select DAI stablecoin as the payment cryptocurrency.
+3. Add a new client using the email address invoice@scrf.io.
+4. Complete further details with the following information:
+```
+  Smart Contract Research Forum
+  1250 Broadway
+  36th Fl
+  10001 New York
+```
```
Comment (ghettodev) [inline en/content_how_to_get_compensated_for_work.md:29]: Its probably better to us quotes instead of code for this. 

i.e. 

`>` not back ticks

**Review by ghettodev — DISMISSED** [review]

**Review by vsterling — COMMENTED** [review]
Implemented the two suggestions to remove the ampersand and add a quote for the address.

**Review by SirAh0 — APPROVED** [review]
looks good! compared against google doc source

**Pull request merged**